docs/command_line_reference.rst

```diff
@@ -710,6 +710,13 @@ The following invocation will list the 50 most recent races::
 
 Suppresses some output on the command line.
 
+``kill-running-processes``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only one Rally benchmark is allowed to run at the same time. If any processes is running, it is going to kill them and allow Rally to continue to run a new benchmark.
```

Comment:
I think we could add an explanation why we want that (in order to ensure that benchmark results are not skewed due to unintentionally running multiple benchmarks at the same time).

Reply:
Added more information in the docs about why we want that. Can you check again?